Edited the section on command-line quickstart (#997)
ipfs/ipfs-docs: docs/how-to/command-line-quick-start.md

![Web console connection view](./images/command-line-quick-start/webui-connection.png)

> The web console shows files that are in your [Mutable File System (MFS)](https://docs.ipfs.io/concepts/file-systems/#mutable-file-system-mfs) which is a tool built into the web console that helps you navigate IPFS files in the same way you would a name-based file system.

Reviewer: ```suggestion
The web console shows files that are in your [Mutable File System (MFS)](../concepts/file-systems#mutable-file-system-mfs). MFS is a tool built into the web console that helps you navigate IPFS files in the same way you would a name-based file system.
```

Links need to be relative to the rest of the repo.

> in order to get ipfs files to show up in the web console (including ones you [added using the CLI](https://docs.ipfs.io/reference/cli/#ipfs-add)), copy them over to MFS:

Reviewer: ```suggestion
When you add files using the [CLI command `ipfs add ...`](../cli#ipfs-add), these files are not automatically available within the MFS. To view files in IPFS Desktop that you added using the CLI, you must copy the files over to the MFS:
```

Author: I think it's worth having some kind of context here for MFS-- let users know within this that MFS is the thing that lets people know it is what the IPFS uses and that it is separate from the rest of the IPFS docs

> in order to get ipfs files to show up in the web console (including ones you [added using the CLI](https://docs.ipfs.io/reference/cli/#ipfs-add)), copy them over to MFS:

Author: ```suggestion
in order to get ipfs files to show up in the web console (including ones you [added using the CLI](https://docs.ipfs.io/reference/cli/#ipfs-add)), copy them over to MFS, which is a tool built into the IPFS client used for navigating IPFS files:
```

> ```
> ipfs files cp /ipfs/<ipfs-CID>
> ```

Reviewer: ````suggestion
```shell
ipfs files cp /ipfs/<ipfs-CID>
```
````

Code blocks must have a language assigned to them. If there isn't a relevant language, use ```plaintext.
